css/style.css

```diff
@@ -1,4 +1,207 @@
 .body {
   font-family: 'Montserrat', sans-serif;
   font-weight: normal;
+  max-width: 1440px;
+  min-width: 320px;
+  overflow-x: hidden;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  padding-top: 24px;
+  padding-bottom: 72px;
+}
+
+.header__logo-img {
+  margin-right: 258px;
+  padding-left: 43px;
+}
+
+.menubar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.menubar__link {
+  text-decoration: none;
+  font-weight: 600;
+  font-size: 15px;
+  color: black;
+  white-space: nowrap;
+}
+
+.menubar__link:hover, 
+.menubar__link:active, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)